test/Microsoft.Azure.WebJobs.Host.TestCommon/TestHelpers.cs

```diff
@@ -121,8 +121,7 @@ public static IHostBuilder ConfigureDefaultTestHost(this IHostBuilder builder, p
                    // Register this to fail a test if a background exception is thrown
                    services.AddSingleton<IWebJobsExceptionHandlerFactory, TestExceptionHandlerFactory>();
                })
-               .ConfigureTestLogger()
-               .AddAzureStorage();
+               .ConfigureTestLogger();
```

Comment:
Nice, I was going to suggest this as well -- it's easy enough to call `AddAzureStorage()`  when you need it, but hard for tests to remove it.